src/gcp/eventarc.ts

```diff
@@ -0,0 +1,95 @@
+import { Client } from "../apiv2";
+import { eventarcOrigin } from "../api";
+import { last } from "lodash";
+import { fieldMasks } from "./proto";
+
+export const API_VERSION = "v1";
+
+export interface Channel {
+  /** User provided name. Must be projects/p/locations/l/channels/c */
```

Comment:
Hm, in that case, is this even information the 3P developer needs?  

Is the user-specified name the one they handle in their code?  Do they ever see the after-resolution name at all?

Reply:
I think the 3P developer will only need their user-specified name. As far as I can tell, the full channel name is only visible via the Eventarc console. Do you think the comment is unnecessary?

Reply:
I'd say the detail about the name is only necessary if we expect the user to view it in the Eventarc console.  If not, then it's unnecessary and potentially confusing.

It's still breaking my brain that you can create multiple user-defined names for multiple channels (true?) but they all resolve to one single specific name/path, but I don't always understand everything :)

Reply:
I can see how the comment would cause confusion for 3p developers, so I would be okay with removing it. 

The channel resources are associated with a project and location, so I think the goal was to keep it simple for 3p developers by providing reasonable defaults for project + location. I think the average use case is simple enough that we won't see too much confusion about channel naming conventions (since they don't really have to worry about anything but providing a channel ID), but maybe we will get some feedback about this as the trigger becomes more popular...

Reply:
Oh, OK, I get it now -- p and location (or some or all of that path) are variables.  

Nonetheless, I like the plan of leaving out the detail unless we see demand for it.  Thanks!